dialogue/TownNPCs/Mocha/RandomAzura/Azura_Mocha_Random1.json

```diff
@@ -22,12 +22,20 @@
       "¡No hay nada que yo no sepa!",
       "<w>Ehhh,</w> debería de volver al trabajo."
     ],
+    "pl": [
+      "Ty wiesz, @CharacterName...\nLepiej nie zostawiać swoje drzwi otwarte na zbyt długo.",
```

Comment:
"Lepiej nie zostawiać swoich drzwi otwartych zbyt długo" will be grammatically correct